### DISCUSSION TITLE
Connect Table column not showing data.

### DISCUSSION THREAD

**Discussion by naodya** [AntlerVC/firetable#1000] [Support / Q&A] [answered]
Hi there, I have a Connect Table column, I managed to point to the correct collection and populate the list. However once I select a value, the value is not showing up in the cell, see screenshot below. I'm I missing something? And yes Algolia is wired and record is being indexed.

![2022-12-06 at 12 07 PM](https://user-images.githubusercontent.com/1115720/205977444-6f829374-268b-46ba-a06b-1322c73936b9.png)

**Comment by BeeBombshell** [comment, marked as answer]
Hey @naodya. I tried replicating your issue. However, I couldn't match the exact conditions that you're facing. It would be helpful if you could provide a screenshot of your entire derivative function (along with the label formatter). 

Also, I'm attaching a screenshot to display the JSON object returned in the console log of the browser. Here, as you can see the `hits` array contains data for the options in the connector. You could also attach a similar screenshot so that I can get to the bottom of this issue.
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/53828745/210820734-cbceb8f0-72f4-45aa-a20b-881994388419.png">

**Reply by BeeBombshell** [reply]
Hey @naodya, closing this discussion since it has been unattended for a long time. If you're still facing this issue, feel free to create a new one.
